src/VisualStudio/IntegrationTest/IntegrationTests/AbstractEditorTest.cs

```diff
@@ -78,5 +73,258 @@ protected void SetUpEditor(string markupCode)
                 VisualStudioWorkspaceOutOfProc.SetPrettyListing(LanguageName, originalValue);
             }
         }
+
```

Comment:
Thank you! Removed all methods below SetUpEditor because they are already in extension methods.